src/parser.cc

```diff
@@ -1115,7 +1115,8 @@ bool readXml(tinyxml2::XMLElement *_xml, ElementPtr _sdf, Errors &_errors)
         }
       }
 
-      if (descCounter == _sdf->GetElementDescriptionCount())
+      if (descCounter == _sdf->GetElementDescriptionCount()
+            && std::strchr(elemXml->Value(), ':') == nullptr)
       {
         sdfdbg << "XML Element[" << elemXml->Value()
```

Comment:
we should consider promoting this to a visible console message in libsdformat11

Reply:
https://github.com/osrf/sdformat/issues/409